zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -410,7 +425,12 @@ private static ProcessDefinitionFilter toProcessDefinitionFilter(
   }
 
   private static OffsetDateTime toOffsetDateTime(final String text) {
-    return StringUtils.isEmpty(text) ? null : OffsetDateTime.parse(text);
+    try {
+      return StringUtils.isEmpty(text) ? null : OffsetDateTime.parse(text);
+    } catch (final Exception e) {
+      final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX");
+      return OffsetDateTime.parse(text, formatter);
```

Comment:
❓ Why do we need to add a custom formatter here?

Reply:
In [current Usage Metrics documentation](https://docs.camunda.io/docs/next/self-managed/operate-deployment/usage-metrics/) the format `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`
is mentioned. This format is not supported in `OffsetDateTime::parse`

One way would be to adjust the documentation.

Reply:
Reverting this into its former state